pinot-common/src/main/java/org/apache/pinot/common/utils/TlsUtils.java

```diff
@@ -60,6 +61,8 @@
   private static final String TRUSTSTORE_PASSWORD = "truststore.password";
   private static final String SSL_PROVIDER = "ssl.provider";
 
+  private static SSLContext _sslContext = null;
```

Comment:
Just curious about the naming convention for static variables - I'd thought they were always upper-cased.